tests/unit/requirements.txt

```diff
@@ -0,0 +1,5 @@
+mock
```

Comment:
Why we are using mock, `unittest.mock` is not enough?

Reply:
Actually `unittest.mock` is enough. I've made the changes.